usr/share/rear/layout/save/GNU/Linux/20_partition_layout.sh

```diff
@@ -249,15 +249,20 @@ Log "Saving disk partitions."
             devname=$(get_device_name $disk)
             devsize=$(get_disk_size ${disk#/sys/block/})
 
-            disktype=$(parted -s $devname print | grep -E "Partition Table|Disk label" | cut -d ":" -f "2" | tr -d " ")
+            #Check if blockd is a path of a multipath device.
+            if multipath -c /dev/${blockd} > /dev/null 2>&1 ; then
```

Comment:
This implies that `multipath` is a required binary for rear! In lots of set-ups multipath is not used. Is there no other way to make this test?